jablib/src/test/java/org/jabref/logic/quality/consistency/BibliographyConsistencyCheckResultTxtWriterTest.java

```diff
@@ -142,7 +142,7 @@ void checkDifferentOutputSymbols(@TempDir Path tempDir) throws IOException {
         assertEquals("""
                 Field Presence Consistency Check Result
 
-                | entry type | citation key | Custom | Pages | Title |
+                | entry type | citation key | custom | Pages | Title |
```

Comment:
Discussion: I think, we should use the field name instead of the display name --> follow-up for someone else :)